packages/multi-parser/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@asyncapi/multi-parser",
+  "version": "2.0.0",
+  "description": "This tool allows to parse AsyncAPI documents and produce a desired interface based on a given Parser-API version",
+"bugs": {
+    "url": "https://github.com/asyncapi/parser-js/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/asyncapi/parser-js.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/asyncapi/parser-js",
+  "sideEffects": false,
+  "main": "cjs/index.js",
+  "module": "esm/index.js",
+  "types": "esm/index.d.ts",
+  "files": [
+    "/esm",
+    "/cjs",
+    "LICENSE",
+    "README.md"
+  ],
+  "scripts": {
+    "build": "npm run build:esm && npm run build:cjs",
+    "build:esm": "tsc",
+    "build:cjs": "tsc --project ./tsconfig.cjs.json",
+    "test": "npm run test:unit",
+    "test:unit": "cross-env CI=true jest --coverage",
+    "lint": "eslint --max-warnings 0 --config ../../.eslintrc  --ignore-path ../../.eslintignore .",
+    "lint:fix": "eslint --max-warnings 0 --config .../../eslintrc  --ignore-path ../../.eslintignore . --fix",
+    "generate:readme:toc": "markdown-toc -i \"../../README.md\"",
+    "generate:assets": "npm run generate:readme:toc",
+    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
+    "prepublishOnly": "npm run generate:assets"
+  },
+  "dependencies": {
+    "@asyncapi/avro-schema-parser": "^3.0.3",
+    "@asyncapi/openapi-schema-parser": "^3.0.4",
+    "@asyncapi/protobuf-schema-parser": "^3.0.0",
+    "@asyncapi/raml-dt-schema-parser": "^4.0.4",
+    "parserapiv1": "npm:@asyncapi/parser@^2.1.0",
+    "parserapiv2": "npm:@asyncapi/parser@3.0.0-next-major-spec.8"
```

Comment:
Where is the `parserapiv3` ?

Reply:
yes this is shifted to line number 65. as "@asyncapi/parser": "file:../parser"
which will take the latest version as a  @asyncapi/parser

Reply:
Actually I was thinking as the parser is in the local now will not need to keep it as dependency. but now i am thinking though the test are passing successfully it should be in dependency only. anyway updated the added it back.

Reply:
Actually I was thinking as the parser is in the local now will not need to keep it as dependency. but now i am thinking though the test are passing successfully it should be in dependency only. anyway updated the added it back.